tools/run_tests/run_performance_tests.py

```diff
@@ -238,6 +238,9 @@ def start_qpsworkers(languages, worker_hosts):
 def create_scenarios(languages, workers_by_lang, remote_host=None, regex='.*',
                      bq_result_table=None):
   """Create jobspecs for scenarios to run."""
+  all_workers = [worker
```

Comment:
When we mangle the workers to support cross language servers, we ended up loosing track of a python server.